tests/test_webclient.py

```diff
@@ -414,6 +417,8 @@ def testPayload(self):
         ).addCallback(self.assertEqual, to_bytes(s))
 
     def testPayloadDisabledCipher(self):
+        if sys.implementation.name == "pypy" and parse_version(cryptography.__version__) <= parse_version("2.3.1"):
+            self.skipTest("This does work in PyPy with cryptography<=2.3.1")
```

Comment:
Well this is awkward, I don't really know why this happens :shrug:

Reply:
Is a “not” missing in the skip message?

As for the reason, it would be ideal to find it out and mention it. I’ve found https://github.com/pyca/cryptography/commit/ce6fe07632c17f16221f02ad19c6875327cdd10a , but it was not introduced in 2.3.1, but in 2.3. Could it be the cause? It seems like it would cause things to break starting with 2.3, not to be fixed starting from 2.3.1.

The only change specific to 2.3.1 was [the rebuild of wheels with a newer OpenSSL version](https://github.com/pyca/cryptography/blob/master/CHANGELOG.rst#231---2018-08-14): 1.1.0g → 1.1.0h. From the [OpenSSL changelog](https://www.openssl.org/news/openssl-1.1.0-notes.html) I cannot tell much, but if we are sure that 2.3.1 is the point where things change, I guess we can assume that the new wheels are what fixes things, and we can mention that in the test.